test/src/examples_complete_test.go

```diff
@@ -24,7 +24,7 @@ func TestExamplesComplete(t *testing.T) {
 		TerraformDir: "../../examples/complete",
 		Upgrade:      true,
 		// Variables to pass to our Terraform code using -var-file options
-		VarFiles: []string{"fixtures.us-east-2.tfvars"},
+		VarFiles: []string{"fixtures.us-east-2.tfvars","fixtures.us-east-2.spot.tfvars"},
```

Comment:
Instead of expanding on VarFiles, it would be best to create a separate unit test otherwise one of these inputs can stomp over the other inputs.

Reply:
cc: @cloudposse/engineering for thoughts here

Reply:
does this commit [c34444c](https://github.com/cloudposse/terraform-aws-ec2-instance/pull/202/commits/c34444c037cd6e3151055a5cfbda78ac05298326) apply to this condition?

a new function  `TestSpotInstanceComplete` is added to test the EC2 spot instance